nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="node-maintenance-operator"]
+= Using the Node Maintenance Operator to place nodes in maintenance mode
+include::modules/common-attributes.adoc[]
+:context: node-maintenance-operator
+toc::[]
+
+You can use the Node Maintenance Operator to place nodes under maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of OpenShift Virtualization installation.
+
+[NOTE]
+====
+If you have installed OpenShift Virtualization, you must use the Node Maintenance Operator that is bundled with it. 
+====
+
+include::modules/eco-about-node-maintenance-standalone.adoc[leveloffset=+1]
+
+include::modules/eco-maintaining-bare-metal-nodes.adoc[leveloffset=+1]
+
+You can install the Node Maintenance Operator using the web console or the OpenShift CLI (`oc`).
```

Comment:
No instructions are needed for how to install?

Reply:
Hi Or, as I have noted in the comment, the installation instructions are in progress

> Hi @razo7 @anna-savina Kindly review the documentation for the standalone NMO. Please note that I am working on the installation sections and I will submit them in this PR later today. In the interim, it would help to get reviews on what we have so far. Thank you!
> 
> cc: @beekhof

Reply:
@razo7 I have now added the installation from web and cli sections